.github/workflows/build_publish_docker_image.yml

```diff
@@ -0,0 +1,25 @@
+name: Build and publish develop image
+
+on:
+    push:
+      branches:
+        - '*'
+      tags:
+        - '*'
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        name: Build image job
+        steps:
+            - name: Checkout master
+              uses: actions/checkout@master
+            - name: Build and publish image
+              uses: ilteoood/docker_buildx@master
+              with:
+                tag: latest,beta
```

Comment:
How does the tagging work?

Currently we use
- `beta` meaning latest master build
- `latest` meaning latest stable build - this is a build with a git tag on
- `1`  latest in the 1 series
- `1.52`  latest in the 1.52 series
- `1.52.0`  actual version number

So we need to do something different for building betas vs building the `latest`.

Maybe tagging the latest could be a different step?

Reply:
This pipeline work as follow: everytime you push or create a release tag, a new version is uploaded to docker hub with tag latest and beta.

Can I assume that everytime that you push on master, I can upload with beta tag?
The rest of the tag should be added only on the creation of a new release. Right?

Reply:
I think you got the logic correct here :-) Commits to master make a beta tag, the rest are on the creation of a new release.